demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -120,14 +119,9 @@ def update_checked_flags_by_support_level(self, file_path):
             metadata_path = os.path.join(PACKS_DIR, pack_name, PACKS_PACK_META_FILE_NAME)
             metadata_json = self.get_metadata_file_content(metadata_path)
             support = metadata_json.get(PACK_METADATA_SUPPORT)
-            certification = metadata_json.get(PACK_METADATA_CERTIFICATION)
 
-            if support == 'partner':
-                if certification is not None and certification != 'certified':
-                    self.add_flag_to_ignore_list(file_path, 'non-certified-partner')
-
-            elif support == 'community':
-                self.add_flag_to_ignore_list(file_path, 'community')
```

Comment:
Why did you remove this?

Reply:
no need for the elif case, packed all in the if case

Reply:
True, missed the 'community' reference. 👍